packages/consts/src/consts.ts

```diff
@@ -278,20 +278,22 @@ export const APIFY_ENV_VARS = {
     INPUT_SECRETS_PRIVATE_KEY_PASSPHRASE: 'APIFY_INPUT_SECRETS_PRIVATE_KEY_PASSPHRASE',
     IS_AT_HOME: 'APIFY_IS_AT_HOME',
     LOCAL_STORAGE_DIR: 'APIFY_LOCAL_STORAGE_DIR',
-    LOG_LEVEL: 'APIFY_LOG_LEVEL',
     LOG_FORMAT: 'APIFY_LOG_FORMAT',
-    META_ORIGIN: 'APIFY_META_ORIGIN',
+    LOG_LEVEL: 'APIFY_LOG_LEVEL',
     METAMORPH_AFTER_SLEEP_MILLIS: 'APIFY_METAMORPH_AFTER_SLEEP_MILLIS',
+    META_ORIGIN: 'APIFY_META_ORIGIN',
     PERSIST_STATE_INTERVAL_MILLIS: 'APIFY_PERSIST_STATE_INTERVAL_MILLIS',
-    PROXY_PASSWORD: 'APIFY_PROXY_PASSWORD',
     PROXY_HOSTNAME: 'APIFY_PROXY_HOSTNAME',
+    PROXY_PASSWORD: 'APIFY_PROXY_PASSWORD',
     PROXY_PORT: 'APIFY_PROXY_PORT',
     PROXY_STATUS_URL: 'APIFY_PROXY_STATUS_URL',
     PURGE_ON_START: 'APIFY_PURGE_ON_START',
     SDK_LATEST_VERSION: 'APIFY_SDK_LATEST_VERSION',
     SYSTEM_INFO_INTERVAL_MILLIS: 'APIFY_SYSTEM_INFO_INTERVAL_MILLIS',
     TOKEN: 'APIFY_TOKEN',
     USER_ID: 'APIFY_USER_ID',
+    USER_IS_PAYING: 'APIFY_USER_IS_PAYING',
```

Comment:
How does this relate to API permissions @tobice pls check this as well

Reply:
Actually pretty well... it's the same situation as with the proxy password. This eliminates the need for Actors to fetch it from API, and for us to create a scope for it :thinking: